ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -148,6 +153,9 @@ message Right {
         string party = 1;
     }
 
+    // The right to administrate the identity provider.
+    message IdentityProviderAdmin {}
+
     // Required
     oneof kind {
         // The user can administrate the participant node.
```

Comment:
Is that right scoped to identity provision in some form? If yes, please clarify how in the comment.

Reply:
Made an attempt :)